quark/cli.py

```diff
@@ -372,6 +380,24 @@ def entry_point(
         with open(output, "w") as file:
             json.dump(json_report, file, indent=4)
             file.close()
+        
+    # Generate web report
+    if webreport:
+        if isinstance(data, ParallelQuark):
+            data.apply_rules(rule_buffer_list)
```

Comment:
I suggest removing the method call because we don't need it to generate reports. The method informs the `ParallelQuark` module to prepare for analyzing the given rule.